emily/handler/Cargo.toml

```diff
@@ -6,9 +6,11 @@ edition = "2021"
 [[bin]]
 name = "emily-handler"
 
+[features]
 default = ["testing"]
 integration-tests = ["testing"]
 testing = ["fake"]
+emily = ["integration-tests"]
```

Comment:
I think this can be removed. It doesn't seem to be used anywhere.

Reply:
Done.